source/gui/__init__.py

```diff
@@ -707,7 +707,7 @@ def _appendHelpSubMenu(self, frame: MainFrame) -> None:
 
 			if updateCheck:
 				# Translators: The label of a menu item to manually check for an updated version of NVDA.
-				item = self.menu.Append(wx.ID_ANY, _("&Check for update..."))
+				item = self.menu.Append(wx.ID_ANY, _("Check for &update..."))

```

Comment:
Can you please update the PR description and title to reflect the intended change, to keep "check for update" in the help menu

```suggestion
				item = menu_help.Append(wx.ID_ANY, _("&Check for update..."))
```